@here/harp-mapview/lib/VisibleTileSet.ts

```diff
@@ -521,9 +521,6 @@ export class VisibleTileSet {
                 i++
             ) {
                 const tileEntry = visibleTileKeys[i];
-                if (!dataSource.shouldRender(displayZoomLevel, tileEntry.tileKey)) {
```

Comment:
Why was this check removed? Aren't we requesting many more tiles now?

Reply:
The check was a duplicate. The tile keys in visibleTileKeys were already checked with `dataSource.shouldRender` in `getVisibleTileKeysForDataSources()`.